docs/NSR.md

```diff
@@ -219,7 +184,9 @@ CA Infrastructure MUST be in a Physically Secure Environment.
 
 ##### 1.2.2
 
-Connections to and within the CA Infrastructure MUST be authenticated and encrypted except OCSP and CRL.
+Connections to the CA Infrastructure MUST be authenticated and encrypted, except where documented that a formal specification prohibits or limits the use of authentication and/or encryption.
```

Comment:
Maybe this should say ", except where a formal specification prohibits ..."?  It seems that "where documented that" adds an unnecessary record-creating requirement and could be deleted without changing meaning. But I'm open to other suggestions/comments.